src/appwrite.js

```diff
@@ -104,27 +104,30 @@ export const api = {
             [],
             limit,
             offset,
-            "created_at",
-            "DESC",
-            "int"
+            undefined,
+            "after",
```

Comment:
undefined is the cursor, and after is cursor direction? Can we make them both undefined?